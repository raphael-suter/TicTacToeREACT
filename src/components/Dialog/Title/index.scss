@import "../../../styles/colors.scss";

.title {
  display: block;
  margin: 0.6rem 0.6rem 0.2rem;
  font-size: 30px;
}
