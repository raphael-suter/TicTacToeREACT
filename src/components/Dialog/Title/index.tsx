import React from 'react';
import './index.scss';

interface Props {
  text: string;
}

export const Title = ({ text }: Props) => (
  <span className='title'>{text}</span>
);
