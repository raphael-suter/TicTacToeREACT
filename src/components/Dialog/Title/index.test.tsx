import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import React from 'react';
import Title from '.';

describe('Title', () => {
  test('Renders correctly.', () => {
    const wrapper = shallow(<Title text={''}/>);
  });
});
