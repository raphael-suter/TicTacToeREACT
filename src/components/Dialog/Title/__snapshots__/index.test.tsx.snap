// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title Renders correctly. 1`] = `<styled.h1 />`;
